### PR TITLE
Ossmc regression extended

### PR DESCRIPTION
**Pull request by ScriptingShrimp**
smoke testing for OCP workloads > deployments for presence of service mesh elements 

additional refactoring and naming conventions 

tested on OCP all passing

### CODE REVIEW

File: plugin/cypress/e2e/featureFiles/workloads-deployments.feature
Comment (ferhoyos) [inline plugin/cypress/e2e/featureFiles/workloads-deployments.feature:16]:
```suggestion
        When user clicks tab with "Overview" button
        Then user is able to see WorkloadDescriptionCard with Kiali Workload
```
Proper action here is `When`

File: plugin/cypress/e2e/featureFiles/workloads-deployments.feature
Comment (ferhoyos) [inline plugin/cypress/e2e/featureFiles/workloads-deployments.feature:19]:
```suggestion
        When user clicks tab with "Traffic" button
```
Proper action here is `When`

File: plugin/cypress/e2e/workloads-deployments.ts
Comment (ferhoyos) [inline plugin/cypress/e2e/workloads-deployments.ts:19]:
```suggestion
Then('user is able to see WorkloadDescriptionCard with Kiali Workload', () => {
```
Proper action here is `Then`

File: plugin/cypress/e2e/workloads-deployments.ts
Comment (ferhoyos) [inline plugin/cypress/e2e/workloads-deployments.ts:27]:
```suggestion
Then('user sees {string} dropdown', (dropdownText:string) => {
```
Proper action here is `Then`

**Review by ferhoyos — CHANGES_REQUESTED** [review]
I have not tested it but code LGTM, just small comments.

**Review by nrfox — COMMENTED** [review]
LGTM but I'll let @ferhoyos approve since he's more familiar with OSSMC

**Review by ferhoyos — APPROVED** [review]
LGTM. They are additional steps to be added when bookinfo application can be installed on tests (https://github.com/kiali/openshift-servicemesh-plugin/pull/245), but this could be done as a separate PR.

**Pull request merged**
